nipoppy/extractors/maget_brain/prepare_data.py

```diff
@@ -50,15 +50,17 @@ def get_masked_image(img_path, mask_path, masked_img_path):
 fmriprep_dir = f"{DATASET_ROOT}/derivatives/fmriprep/{fmriprep_version}/output/"
 maget_dir = f"{DATASET_ROOT}/derivatives/maget_brain/{maget_version}/output/"
 maget_preproc_T1w_nii_dir = f"{maget_dir}/ses-{session_id}/preproc_T1w_nii/"
+maget_proc_list_file = f"{maget_preproc_T1w_nii_dir}proc_participant.csv"
```

Comment:
This script is writing a CSV file of participants to process? Who have been processed?